src/test/java/com/google/api/codegen/gapic/testdata/go/go_library.baseline

```diff
@@ -777,8 +776,7 @@ func (c *LibClient) DiscussBook(ctx context.Context, opts ...gax.CallOption) (li
 
 // MonologAboutBook test client streaming.
 func (c *LibClient) MonologAboutBook(ctx context.Context, opts ...gax.CallOption) (librarypb.LibraryService_MonologAboutBookClient, error) {
-    md := metadata.Pairs("x-goog-request-params", fmt.Sprintf("%s=%v", "name", req.GetName()))
```

Comment:
This code had been broken because it expects a variable named `req`, which is not in the method arguments.